api/routes/multiband_image.go

```diff
@@ -24,21 +25,33 @@ import (
 	"net/http"
 	"path"
 	"strconv"
+	"strings"
 )
 
 const (
 	// ThumbnailDimensions is hard coded thumbnail dimension -- could be refactored to be default if we want client to dictate size.
 	ThumbnailDimensions = 125
 )
 
+func getOptions(requestURI string) string {
+	idx := strings.LastIndex(requestURI, "/") + 1 // exclusive
+	return requestURI[idx:]
+}
+
 // MultiBandImageHandler fetches individual band images and combines them into a single RGB image using the supplied mapping.
 func MultiBandImageHandler(ctor api.MetadataStorageCtor) func(http.ResponseWriter, *http.Request) {
 	return func(w http.ResponseWriter, r *http.Request) {
 		dataset := pat.Param(r, "dataset")
 		imageID := pat.Param(r, "image-id")
 		bandCombo := pat.Param(r, "band-combination")
+		paramOption := getOptions(r.URL.Path)
 		isThumbnail, err := strconv.ParseBool(pat.Param(r, "is-thumbnail"))
 		imageScale := util.ImageScale{}
+		if err != nil {
```

Comment:
This should be put after the line that could error. Either the strconv call should be moved down or this moved up one.